tab_sponsorship.md

```diff
@@ -41,6 +41,8 @@ As it turns out, writing a book to a professional standard is a challenging task
 
 100% of the funds raised go directly into the project budget and will be used to fund production of the final release, including:
 
-Editing and proofreading by professional editors
-Graphic design and layout
-Any leftover funds will be donated to the OWASP Foundation to the mobile security project for future use.
+Editing and proofreading by professional editors Graphic design and layout. Any leftover funds will be donated to the OWASP Foundation to the mobile security project for future use.
```

Comment:
Why removing the list?

```suggestion
Editing and proofreading by professional editors as well as graphic design and layout improvements. Any leftover funds will be donated to the OWASP Foundation to the mobile security project for future use.
```

Reply:
so i can put it back now :D , you are right, lists are better.